tests/beaker/tests/create_lv_with_param_alloc.rb

```diff
@@ -0,0 +1,82 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96574 - create logical volume without parameter 'alloc'"
```

Comment:
Title of the Beaker test does not match the title of the corresponding TestRail test case. Having them match acts as a second factor in verify that the correct test cases were automated.